src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/activity/ScheduleReductionExpirationActivity.java

```diff
@@ -0,0 +1,12 @@
+package it.gov.pagopa.pu.workflow.wf.debtposition.custom.activity;
+
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+import it.gov.pagopa.payhub.activities.dto.debtposition.syncwfconfig.FineWfExecutionConfig;
+
+@ActivityInterface
+public interface ScheduleReductionExpirationActivity {
+
+  @ActivityMethod
+  String expireFineReduction(Long debtPositionId, FineWfExecutionConfig executionParams);
```

Comment:
This will not expire, it will schedule the expiration WF
```suggestion
  String scheduleExpireFineReduction(Long debtPositionId, FineWfExecutionConfig executionParams);
```